runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -55,6 +55,7 @@
 import net.runelite.api.events.GameStateChanged;
 import net.runelite.api.events.GameTick;
 import net.runelite.api.events.NpcDespawned;
+import lombok.extern.slf4j.Slf4j;
```

Comment:
why is this moved from its original position?

Reply:
Most likely just a "mistake" made,
moved it back to origin position.